content/applications/finance/payment_providers/razorpay.rst

```diff
@@ -2,53 +2,86 @@
 Razorpay
 ========
 
-`Razorpay <https://razorpay.com/>`_ is an online payments provider established in India and
-covering more than 100 payment methods.
+`Razorpay <https://razorpay.com/>`_ is an online payments provider established in India thatsupports
+more than 100 payment methods.
+
+.. _payment_providers/razorpay/oauth-connection:
+
+Create a Razorpay account with Odoo (Indian companies only)
+===========================================================
+
+.. note::
+   - This method is only available for Indian companies.
+   - If you want to :ref:`try Razorpay as a test <payment_providers/test-mode>` first, use the
+     :ref:`manual credentials and webhook input method <payment_providers/razorpay/manual-connection>`.
```

Comment:
I am not sure this note is quite useful instead we could just give warning `Connect flow does not support test mode` (or don't add anything at all) because we don't want user to goto lengthy onboarding process and for first time user this is not very useful and user can simply test it on like account by making 1INR payment

Still don't have very strong opinion on that. may be @AntoineVDV can help us here? 🙈